ethereum/datastructures/build.gradle

```diff
@@ -7,6 +7,7 @@ jar {
 }
 
 dependencies {
+  compile group: 'org.mapdb', name: 'mapdb', version: '3.0.7'
```

Comment:
Could we keep versions centralised in `gradle/versions.gradle`?  So this would just be:
```compile 'org.mapdb:mapdb'```

and in `versions.gradle`:
```dependency 'org.mapdb:mapdb:3.0.7'```